rules/how-to-monetize-apps/rule.md

```diff
@@ -14,66 +16,97 @@ redirects:
+    
+One-time fees work with either paid apps or 'freemium' apps; users can either buy the app up-front or use the app for free, and pay to unlock premium features.
 
-
-
-::: bad
-Bad example: paid with currency  
-:::
-
+### Subscription fee
+Subscriptions have become much more popular, even for traditional desktop software. Rather than pay a one-time fee, users pay a recurring monthly or annual fee to use the software. This might be the right approach for you if:
+* Your product is a SaaS product
+* Your product is likely to get recurring regular use *without* you needing to continually drive engagement (e.g. An essential, or at least high value, productivity tool)
+* Your product has other ongoing maintenance costs (e.g. Content development)
 
 ::: good
-Good example: paid with abstract currency  
+![Figure: Good example - Microsoft 365 is available to home and business users for a low recurring cost](microsoft365.png)
 :::
 
+[Microsoft 365](https://www.microsoft.com/microsoft-365) (formerly Office 365) is a suite of productivity tools that run on mobile, desktop and web. With a subscription, users get access to these apps and the connected services that tie them together in the cloud, and the cost over a 3 year period (the average time between desktop releases before Office 365) is lower than the cost of a desktop license amortized over the same period.
 
+### Micro-transactions
+Micro-transactions is the practice of continually charging comparatively small amounts of money over a long period of time. Micro-transactions can be a great fit if used responsibly, but are considered unethical when combined with nefarious practices like [dark patterns](https://www.youtube.com/watch?v=lJUW0iZzAaQ), 'pay-to-win' or exploiting gambling addition.
+    
+For these reasons, micro-transactions are [increasingly](https://www.vg247.com/australian-government-rules-games-microtransactions-must-address-packaging) [subject](https://www.gamespot.com/articles/you-might-have-to-prove-your-age-to-buy-a-loot-box/1100-6474443/) to scrutiny from [governments](https://www.aph.gov.au/Parliamentary_Business/Committees/House/Social_Policy_and_Legal_Affairs/Onlineageverification/Report) and [regulators](https://www.gamesindustry.biz/australian-investigation-finds-loot-boxes-are-pscyholgocially-akin-to-gambling). This might be the right approach for you if:
+* You are certain you have an ethical model for micro-transactions
+* You are not exploiting minors or people with a gambling addiction
+* You have a genuine use case for providing small purchases in your app that can bring joy to your users
 
+`youtube: https://www.youtube.com/embed/fKK9nVLvhGM`
 
 ::: bad
-Bad example: treat all customers the same  
+Video: Bad example - Pokemon Go, one of the most successful mobile apps of all time, combines the freemium model with micro-transactions to exploit a vulnerable subset of users. It's incredibly effective, but unethical
```

Comment:
IIRC PKMN Go is relatively OK with Micro-Transactions (But it is a much more well known and relatable IP). That's an awesome video though.

You could honestly make a list of many many games that have horrendous practices around monetising their games.

Diablo Immortal, or even Genshin Impact are a much more egregious and recent example IMO that I can think of. Exploiting those with gambling tendencies. 

[Here's a long and in-depth video around Diablo and its monetisation strategy](https://www.youtube.com/watch?v=o17lBUZgjTs)

Can't add a suggestion for you here :/ But you may prefer to keep Pokemon for how globally known it is, I'm just pointing out some more examples :)